Library/Formula/racer.rb

```diff
@@ -0,0 +1,44 @@
+ class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+  url "https://github.com/phildawes/racer/archive/v1.0.0.tar.gz"
+  sha256 "78895296ed688eeccbaf7745235f0fc503407bfa718f53583a4dcc9e1246b7f5"
+
+  keg_only "You should set path in your special editor which use racer"
+
+  option "without-src", "Do not get rust source for racer from www.rust-lang.org"
+
+  depends_on "rust" => :build
+
+  resource "rustsrc" do
+    url "https://static.rust-lang.org/dist/rustc-1.3.0-src.tar.gz"
+    sha256 "ea02d7bc9e7de5b8be3fe6b37ea9b2bd823f9a532c8e4c47d02f37f24ffa3126"
+  end
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/racer"
+
+    if build.with?("src")
+      resource("rustsrc").stage do
+        (prefix/"rustsrc").install Dir["./src/*"]
+      end
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    Rust source is installed at /usr/local/opt/racer/rustsrc/src by default.
```

Comment:
Homebrew is not always in `/usr/local`. This should be `"#{opt_prefix}/racer/rustsrc/src"`